docs/support/troubleshooting.md

```diff
@@ -29,3 +30,18 @@ You can recover from that issue by identifying the commit in your branch history
 $ git tag -f v<version of the last release> <commit sha1 corresponding to last release>
 $ git push -f --tags origin <your release branch>
 ```
+
+## You do not have permission to publish 'package-name'
+
+When running semantic-release you may encounter the error below:
+
+```
+An error occurred while running semantic-release: { Error: Command failed: npm publish ./. --registry https://registry.npmjs.org/
+npm ERR! publish Failed PUT 403
+npm ERR! code E403
+npm ERR! You do not have permission to publish "package". Are you logged in as the correct user? : package
+```
+
+This has in fact nothing to do with your `NPM_TOKEN` or authentication method. It is actually related to this NPM issue: [https://github.com/npm/npm/issues/8561](https://github.com/npm/npm/issues/8561) and [Improve message thrown from Error code 'E403'](https://github.com/npm/registry/issues/240). The problem is that there is already a package with the same name as the one you are trying to publish. The resolution therefore is to rename your package in your `package.json`
```

Comment:
I wouldn't be that categorical here. There is situation in which this error message is actually correct and caused by a permission. So I would soften the language here.